game-core/src/main/java/games/strategy/util/ExitStatus.java

```diff
@@ -0,0 +1,27 @@
+package games.strategy.util;
+
+/**
+ * A process exit status.
+ */
+public enum ExitStatus {
+  /** The process exited successfully (0). */
+  SUCCESS(0),
+
+  /** The process exited due to a failure (1). */
+  FAILURE(1);
+
+  private final int status;
+
+  private ExitStatus(final int status) {
+    assert status >= 0 : "exit status must be non-negative to avoid signal aliasing";
```

Comment:
Curious why not guava here? Should we convert existing Guava usage to `assert`? Just curious on your take @ssoloff on the topic.

Reply:
In general, precondition violations on API methods (public/protected) are signaled with an exception, while precondition violations on non-API methods (package-private/private) are signaled (when deemed necessary) with an assertion.  [This SO answer](https://stackoverflow.com/a/1957656/3900879) summarizes things nicely.